### PR TITLE
doc: remove mention of image feature dependency

### PR DESCRIPTION
**Pull request by Madhu-1**
avoid specifying the image feature dependencies and add a link to the rbd official document for reference to the image feature dependencies.

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

fixes https://github.com/ceph/ceph-csi/pull/2885#discussion_r808916691

### CODE REVIEW

**Review by Rakshith-R — APPROVED** [review]

File: charts/ceph-csi-rbd/values.yaml [hunk 1]
```diff
@@ -281,8 +281,9 @@ storageClass:
 
   # (optional) RBD image features, CSI creates image with image-format 2 CSI
   # RBD currently supports `layering`, `journaling`, `exclusive-lock`,
-  # `object-map`, `fast-diff`, `deep-flatten` features. If `journaling` is
-  # enabled, must enable `exclusive-lock` too.
+  # `object-map`, `fast-diff`, `deep-flatten` features.
+  # Refer https://docs.ceph.com/en/latest/rbd/rbd-config-ref/#image-features
```
Comment (Yuggupta27) [inline charts/ceph-csi-rbd/values.yaml:285]:
```suggestion
  # Note: Refer https://docs.ceph.com/en/latest/rbd/rbd-config-ref/#image-features
```

**Review by Yuggupta27 — COMMENTED** [review]
Should we add something like `Note` or `Caution` to make sure a user always goes through it and is aware to avoid issues in the future?

File: charts/ceph-csi-rbd/values.yaml (hunk 1 above)
Comment (Madhu-1) [reply charts/ceph-csi-rbd/values.yaml:285]: we dont add note. As this is applicable if they use more image features, as the reference is given IMO should be fine. if the dependencies are not mentioned properly the volume creation will fail with the proper error message.

**Review by Yuggupta27 — APPROVED** [review]

**Review by idryomov — APPROVED** [review]

**Review by humblec — APPROVED** [review]

**Comment by Rakshith-R** [comment]
https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.21/detail/mini-e2e-helm_k8s-1.21/2926/pipeline

jenkins tests were stuck , so added DNM to let https://github.com/ceph/ceph-csi/pull/2911 get merged, 

@mergifyio refresh

**Comment by Rakshith-R** [comment]
@mergifyio rebase

**Pull request merged**